infrastructure/terraform/components/reporting/scripts/create_iceberg_table.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Creates Iceberg table for reporting if it doesn't exist already
+
+ENV=${1:-"no_env"}
+
+if [[ ${ENV} == "no_env" ]]; then
+    echo "Environment name not provided as first argument"
+    exit 1
+fi
+
+table_exists=$(aws glue get-tables --database-name nhs-notify-${ENV}-reporting-database | jq 'any(.TableList[].Name == "'nhs_notify_${ENV}_item_status_iceberg'"; .)')
```

Comment:
The same string derivation of the table name occurs multiple times in this script. Should probably be factored out as a local variable?

Reply:
variable-ised it, thank you